src/timeMachine/components/dynamicFluxFunctionsToolbar/TooltipArguments.tsx

```diff
@@ -19,24 +17,21 @@ class TooltipArguments extends PureComponent<Props> {
       </article>
     )
   }
-
   private get arguments(): JSX.Element | JSX.Element[] {
     const {argsList} = this.props
 
     if (argsList.length > 0) {
-      return argsList.map(a => {
+      return argsList.map(argument => {
+        const description = argument.headline.slice(argument.name.length + 1)
```

Comment:
I was able to confirm all flux parameters' headline property contained the name of the parameter in the beginning of the string. 
for instance: 
fluxParameters": [
{
"name": "url",
**"headline": "url: Opsgenie API URL. Defaults to",**
"required": false,
"description": ""
},
]